### DISCUSSION TITLE
Refresh when clicking the activated tab

### DISCUSSION THREAD

**Discussion by fpronto** [dailydotdev/daily#474] [Feature Requests]
Hi there.

First of all, great work with this project, I really love it.

I want to ask you guys for a small feature. When i have activated the "My Feed" tab and click in that button again it doesn't do anything. It would be cool if it refreshes the feed. My current workaround is to click in the popular tab and my feed again.

This happens because I let my new tab open all the time. I don't know if that isn't the intended way to use it.

Thanks guys

**Comment by idoshamun** [comment]
Hi @fpronto,

The best workaround I can suggest now is to refresh the page.
I'll forward this to the team and see what else we can do.
